src/gululog_repair.erl

```diff
@@ -8,6 +8,9 @@
         , repair_dir/2
         ]).
 
+-export([ copy_file/2
```

Comment:
abstract to common module (gululog_file ?), do it now rather than later.
and as mentioned in QQ, file:rename can be used.